node/sc/bridge_manager.go

```diff
@@ -313,7 +313,7 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 		uri, err = erc721.TokenURI(nil, ev.ValueOrTokenId)
 		if err != nil {
 			if err == vm.ErrExecutionReverted {
-				logger.Warn("The ERC721 token does not support URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
+				logger.Debug("Failed to get an ERC721 URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
```

Comment:
"Failed" seems too strong for `Debug` level log. 
how about changing the message to "Unable to ~" or "Cannot get "?

Reply:
Good. I will reflect. :)